src/components/Login.js

```diff
@@ -1,8 +1,19 @@
 import React, { Component } from "react";
-import { Link } from "react-router-dom";
+import { Link } 
+from "react-router-dom";
```

Comment:
You should set up [prettier](https://github.com/prettier/prettier-vscode) to avoid unnecessary diffs like these.

Reply:
sure